### PR TITLE
WFLY-9388 - There should be replaced stacktrace loggings to stderr with usage of logging

### PR DESCRIPTION
**Pull request by rpelisse**
Upstream Issue: https://issues.jboss.org/browse/WFLY-9388
Issue: https://issues.jboss.org/browse/JBEAP-12220

### CODE REVIEW

File: jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java [hunk 1]
```diff
@@ -131,7 +131,7 @@ public JdrReport collect() throws OperationFailedException {
                 ROOT_LOGGER.debugf(message);
                 skips.append(message);
                 PrintWriter pw = new PrintWriter(new StringWriter());
-                t.printStackTrace(pw);
```
Comment (rsvoboda) [inline jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java]: https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html#printStackTrace-java.io.PrintWriter-

Stack trace is printed to string writer and added to StringBuilder skips

File: jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java [hunk 2]
```diff
@@ -131,7 +131,7 @@ public JdrReport collect() throws OperationFailedException {
...
+                ROOT_LOGGER.error(t.getLocalizedMessage());
```
Comment (rsvoboda) [inline jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java]: Your change is breaking original behavior, you are appending nothing. This change should be reverted

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java (hunk 2 above)
Comment (rpelisse) [reply jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java]: gosh, sorry, I did not noticied the PrintWriter manipulation. I'll fix that right away.

File: connector/src/main/java/org/jboss/as/connector/services/workmanager/WildflyWorkWrapper.java [hunk 3]
```diff
@@ -61,7 +63,7 @@ protected void runWork() throws WorkCompletedException {
                 try {
                     WildflyWorkWrapper.super.runWork();
                 } catch (WorkCompletedException e) {
-                    e.printStackTrace();
+                    ROOT_LOGGER.error(e.getLocalizedMessage());
```
Comment (stuartwdouglas) [inline connector/src/main/java/org/jboss/as/connector/services/workmanager/WildflyWorkWrapper.java]: This should have a meaningful error message (i.e. a message added to the relevant logger), and should include the exception stack trace (like what you have done below).

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]

**Comment by rpelisse** [comment]
@stuartwdouglas good point, I've done so and updated the PR

**Review by stuartwdouglas — APPROVED** [review]

**Comment by rpelisse** [comment]
retest this please

**Pull request merged**